pkg/asset/templates/kubelet.yaml

```diff
@@ -17,27 +17,19 @@ spec:
       - name: kubelet
         image: quay.io/coreos/hyperkube:v1.3.0_coreos.1
         command:
-        - /nsenter
-        - --target=1
-        - --mount
-        - --wd=.
-        - --
-        - ./hyperkube
-        - kubelet
-        - --api-servers={{ index .APIServers 0 }}
-        - --config=/etc/kubernetes/manifests
-        - --allow-privileged
-        - --hostname-override=$(MY_POD_IP)
-        - --cluster-dns=10.3.0.10
-        - --cluster-domain=cluster.local
-        - --kubeconfig=/etc/kubernetes/kubeconfig
-        - --lock-file=/var/run/lock/kubelet.lock
-        - --minimum-container-ttl-duration=3m0s
+        - /bin/bash
+        - -c
+        - "/nsenter --target=1 --mount --wd=. -- ./hyperkube kubelet --hostname-override=$(MY_POD_IP) $(FLAGS)"
         env:
           - name: MY_POD_IP
             valueFrom:
               fieldRef:
                 fieldPath: status.podIP
+          - name: FLAGS
+            valueFrom:
+              configMapKeyRef:
+                name: kubelet-config
```

Comment:
This might present a bit of a problem. This won't allow us to have different configuration for different nodes. For example, "register-unschedulable" or "--node-label=master=true".

The only option there is using the PERSISTENT_KUBELET_OPTS -- but then we're back to defining configuration on disk (which is not updatable via api)